crates/codegen/runtime/cargo/src/runtime/bindings/binding_rules.rs.jinja2

```diff
@@ -0,0 +1,5 @@
+#[allow(clippy::needless_raw_string_hashes)]
+#[allow(dead_code)] // TODO(#982): use to create the graph
+pub const BINDING_RULES_SOURCE: &str = r#####"
```

Comment:
I wonder if we shouldn't use https://doc.rust-lang.org/std/macro.include_str.html ?

Reply:
@AntonyBlakey you mean we "should"?

This PR avoids the additional (identical) copy of the rules file file that we would need in the output crate, only to `include_str!()` it again in the Rust file.